sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -497,9 +496,8 @@ def input_elements(
             raise RuntimeError('Channel closed prematurely.')
           if abort_callback():
             return
-          t, v, tb = self._exc_info
-          if t:
-            raise t(v).with_traceback(tb)
+          if self._exception:
+            raise self._exception
```

Comment:
given that this is within `except queue.Empty`, i think we may want to `raise self._exception from None`

Reply:
see: https://stackoverflow.com/a/18188660/5153670

Reply:
Done.